src/video/win32_v.cpp

```diff
@@ -993,6 +993,25 @@ void VideoDriver_Win32Base::EditBoxLostFocus()
 	SetCandidatePos(this->main_wnd);
 }
 
+std::vector<int> VideoDriver_Win32Base::GetListOfMonitorRefreshRates()
+{
+	std::vector<int> rates = {};
+	EnumDisplayMonitors(nullptr, nullptr, [](HMONITOR hMonitor, HDC hDC, LPRECT rc, LPARAM data) -> BOOL {
```

Comment:
Could this also be a function instead of a lambda? No is a valid answer :D

Reply:
Possibly, but I didn't want taking up more space and instead decided a simple lambda could be sufficient enough.

Reply:
Space we have plenty! Look at there, so many place! :D

Without joking, I don't really have a preference, it just feels a bit large for a lambda. And because you cramped everything in here, it is also hard to read. So it is more that, than anything else :)